main/zoe/guide/README.md

```diff
@@ -61,11 +61,10 @@ that is escrowed with Zoe, Zoe guarantees that the user will either
 get back why they said they wanted, or the user will get back what they
 originally offered.
 
-When a user escrows with Zoe, they get two things back immediately: a
-`seat`, and a JavaScript promise for a future payout. This `seat` has
-methods that the user can call to take action in the smart contract on
-Zoe, without the smart contract ever having access to the underlying
-digital assets. Let's look a particular example to see how this works.
+When a user escrows with Zoe, they get a few things back immediately:
+an `outcome`, and a JavaScript promise for a future payout. The
+`outcome` is the return value of making the offer. Let's look a
```

Comment:
It's actually a promise for the outcome?  Or the outcome is a promise returned by the contract?

Reply:
Yeah, promise for the return value. I'll edit